### PR TITLE
Added Favicon Support for React-Page

### PR DESCRIPTION
**Pull request by ammit**
I used the react-page logo for the favicon, and used connect middleware to include it.

**Favicon -** 
favicon

### CODE REVIEW

**Comment by jordwalke** [comment]
Nice, @ammit! Did you get a chnace to sign the CLA mentioned here (https://github.com/facebook/react/blob/master/CONTRIBUTING.md)?

**Comment by ammit** [comment]
@jordwalke Yes, I did it.

File: server.js [hunk 1]
```diff
@@ -74,6 +74,7 @@ if (!isServer) {
   var app = connect()
     .use(reactMiddleware.provide(buildOptions))
     .use(connect['static'](FILE_SERVE_ROOT))
+    .use(connect.favicon(path.join(FILE_SERVE_ROOT, 'elements/favicon/favicon.ico')))
```
Comment (kassens) [inline server.js]: `/` should not be hardcoded, use `path.join` for everything.

```
path.join(FILE_SERVE_ROOT, 'elements', 'favicon', 'favicon.ico')
```

**Comment by jordwalke** [comment]
@ammit  Merging, now this is looking like a _real_ page!

**Pull request merged**